web/templates/attendee/show.html.eex

```diff
@@ -20,7 +20,7 @@
     <div class="col-sm-6">
       <strong>Email address:</strong> <%= @attendee.email %><br /><br />
       <strong>Phone number:</strong> <%= @attendee.phone |> number_to_phone %><br />
-      <em class="normal-size">* <%= @attendee.sms |> Syndeo.Attendee.display_sms_availability() %></em><br /><br />
+      <em class="normal-size">* <%= @attendee.sms |> display_sms_availability() %></em><br /><br />
```

Comment:
You don't need the `()` on the end here since you're passing in an argument to the function. Doesn't hurt to leave it, just wanted to let you know.